### DISCUSSION TITLE
Incomplete answer: Chapter 4, Problem 17c

### DISCUSSION THREAD

**Discussion by bryzhang** [fifthist/Introduction-To-Probability-Blitzstein-Solutions#34] [General]
The answer lacks an explanation for the Wrigley-Field puzzle. Similar to 16b, you can think of $\frac{m1}{m0}$ as $E(X)$ and $\frac{m2}{m1}$ as $E(X^2)$.

Given the following:
1. Variance is defined as $E(X^2) - E(X)^2$.
2. $\frac{m2}{m1}$ is the children's-view children per family. According to Wrigley-Field, this went down.
3. $\frac{m1}{m0}$ is the family-view children per family. This went up.

Thus, if $E(X^2)$ went down and $E(X)^2$ went up, then the overall variance went down between the two eras. So we can explain the Wrigley-Field problem as a result of decreased variance in children per family.

**Comment by fifthist** [comment]
Hi Bryan, thanks for finding the mistakes, and explaining your solutions. If you choose to do so, feel free to open pull requests with your changes to have them committed. This applies to the other problems you commented on as well.

**Reply by bryzhang** [reply]
I keep telling myself I'll start doing PRs instead. Maybe once I get through this chapter, I'll turn my comments into PRs.
